daemon/container_operations_windows.go

```diff
@@ -3,8 +3,16 @@
 package daemon
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/cloudflare/cfssl/log"
```

Comment:
Yes, I can change this, and update the Unix file as well. I might do it in a separate commit, for separation, but I can definitely keep it in the same PR.